x-pack/test/functional/apps/rollup_job/hybrid_index_pattern.js

```diff
@@ -46,6 +45,11 @@ export default function ({ getService, getPageObjects }) {
       await kibanaServer.uiSettings.replace({
         defaultIndex: 'rollup',
       });
+
+      // The step below is done for the 7.17 ES 8.15 forward compatibility tests
```

Comment:
```suggestion
      // The step below is done for the 7.17 ES 8.x forward compatibility tests
```

[nit] same as mentioned in https://github.com/elastic/kibana/pull/187339#discussion_r1677750930